zstd_vendor/CMakeLists.txt

```
@@ -4,12 +4,15 @@ project(zstd_vendor)
 
 find_package(ament_cmake REQUIRED)
 
+list(INSERT CMAKE_MODULE_PATH 0 "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules")
+
 option(FORCE_BUILD_VENDOR_PKG
   "Build zstd from source, even if system-installed package is available"
   OFF)
 
 if(NOT FORCE_BUILD_VENDOR_PKG)
-  find_package(zstd QUIET)
+  # We need at least VERSION 1.4.4, version check is done in Findzstd.cmake
+  find_package(zstd 1.4.4 QUIET)
```

But the thing is, this requests *exactly* 1.4.4, which is not what we want.  We want at least 1.4.4.  So I still think this should just be:

```suggestion
  find_package(zstd QUIET)
```

And then restore the version check below.

---

Nope. It is at least 1.4.4. I tested it *and* documentation is also on my side: 
```
The [version] argument requests a version with which the package found should be compatible. There are two possible forms in which it may be specified:

A single version with the format major[.minor[.patch[.tweak]]], where each component is a numeric value.

A version range with the format versionMin...[<]versionMax where versionMin and versionMax have the same format and constraints on components being integers as the single version. By default, both end points are included. By specifying <, the upper end point will be excluded. Version ranges are only supported with CMake 3.19 or later.

The EXACT option requests that the version be matched exactly. This option is incompatible with the specification of a version range.
```
From https://cmake.org/cmake/help/latest/command/find_package.html

---

Good point.  I learned something about CMake today.

And there is precedent for this over in https://github.com/ros2/orocos_kdl_vendor/blob/rolling/orocos_kdl_vendor/CMakeLists.txt .  So OK, I'm fine with this change then.